src/App.js

```diff
@@ -19,17 +20,21 @@ export default function App() {
   return (
     <Wrapper>
       <Header>Viary</Header>
+      {currentPage === 'home' && (
+        <StartPage onNavigate={showCameraPage} memories={memories} />
+      )}
       {currentPage === 'camera' && (
         <CameraPage
           onNavigate={showMemoriesPage}
           handleMemorySubmit={handleMemorySubmit}
+          onNavigateBack={showStartPage}
         />
       )}
       {currentPage === 'memories' && (
         <MemoriesPage
           memories={memories}
           onDetail={showDetailMemoryPage}
-          onNavigate={showCameraPage}
+          onNavigateBack={showCameraPage}
```

Comment:
not important but to make it shorter it could be called onBack (instead of onNavigateBack)